nym-vpn-desktop/src-tauri/src/commands/credential.rs

```diff
@@ -0,0 +1,53 @@
+use nym_vpn_lib::{credentials::import_credential_base58, nym_config::defaults::var_names};
+use tauri::{api::path::data_dir, State};
+use tracing::{debug, error, info, instrument};
+
+use crate::{
+    db::{Db, Key},
+    error::{CmdError, CmdErrorSource},
+    fs::util::check_dir,
+    APP_DIR,
+};
+
+const CREDENTIAL_DIR: &str = "credential";
+
+#[instrument(skip_all)]
+#[tauri::command]
+pub async fn add_credential(credential: String, db: State<'_, Db>) -> Result<(), CmdError> {
+    debug!("add_credential");
+    let network_name = std::env::var(var_names::NETWORK_NAME)
+        .inspect_err(|e| error!("failed to get current network name: {}", e))
+        .unwrap_or("unknown".into());
+    let mut path = data_dir().ok_or(CmdError::new(
```

Comment:
it would be good to set the data dir on `NymVpn` as well so we have persistent keys

Reply:
as discussed we'll use _os_data_dir_/nym-vpn/backend/ for backend data dir